cumulusci/tasks/bulkdata/tests/test_mapping_parser.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+import logging
+from io import StringIO
+import pytest
+
+from yaml import safe_load, dump, YAMLError
+
+from cumulusci.tasks.bulkdata.mapping_parser import parse_from_yaml, ValidationError
+
+
+class TestMappingParser:
+    def test_simple_parse(self):
+        base_path = Path(__file__).parent / "mapping_v2.yml"
+        assert parse_from_yaml(base_path)
+
+    def test_after(self):
+        base_path = Path(__file__).parent / "mapping_after.yml"
+        result = parse_from_yaml(base_path)
+
+        step = result["Insert Accounts"]
+        lookups = step["lookups"]
+        assert lookups
+        print(lookups)
```

Comment:
Several `print()` statements in this file. Can we remove those for production?

Reply:
Yes. 3f2336ee